src/pages/manage/manage.html

```diff
@@ -96,11 +101,11 @@
           <ion-card-content>
             <ion-label stacked>Paypal Email Address</ion-label>
             <ion-input
-                    type="email"
-                    placeholder="example@example.com"
-                    required
-                    name="email"
-                    [(ngModel)]="withdrawMoney.email"
+              type="email"
+              placeholder="example@example.com"
+              required
+              name="email"
+              [(ngModel)]="withdrawMoney.email"
             ></ion-input>
```

Comment:
I agree but I only fixed the spacing here. Your suggestions will be implemented in another pull request that fixes the withdraw and deposit logic. Since this pull request is related to changing wallet views when clicking action buttons only.